<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_user_show</class>
 <widget class="QDialog" name="Dialog_user_show">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>111</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>101</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_add_user">
      <property name="text">
       <string>添加新用户</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_load_data">
      <property name="text">
       <string>载入数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_delete_data">
      <property name="text">
       <string>删除用户数据</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_quit">
      <property name="text">
       <string>离开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
